catalog/ui/catalog-ui-filter/src/main/java/org/codice/ddf/catalog/ui/filter/impl/visit/TransformVisitor.java

```diff
@@ -44,7 +47,38 @@
  * <p><i>This code is experimental. While it is functional and tested, it may change or be removed
  * in a future version of the library.</i>
  */
+@SuppressWarnings("squid:S4144" /* Duplication will be addressed by DDF-3926 */)
 public class TransformVisitor<T> extends AbstractFilterVisitor2 {
+  // This extraneous map is an intermediate step and will go away in DDF-3926
+  private static final Map<String, Consumer<FlatFilterBuilder<?>>> OPERATORS =
+      ImmutableMap.<String, Consumer<FlatFilterBuilder<?>>>builder()
+          .put(FilterJson.Ops.NOT, FlatFilterBuilder::not)
+          .put("Not", FlatFilterBuilder::not)
```

Comment:
Might be nice to either make these separate static final in FilterJson.Ops or possibly turn FilterJsonOps into a custom enum with the operator and name.